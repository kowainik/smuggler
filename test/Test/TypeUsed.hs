module Test.TypeUsed where

import Data.Bool (Bool, not)

foo :: Bool
foo = undefined
