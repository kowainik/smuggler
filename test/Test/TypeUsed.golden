module Test.TypeUsed where

import Data.Bool (Bool,)

foo :: Bool
foo = undefined
